Documentation/content/docs/concepts_widgets.md

```diff
@@ -109,18 +109,27 @@ rendered.
 
 ### Scaling representations to be fixed in display space
 
-In order to scale a representation such that it retains the same size in display space,
-it must use the `scaleByDisplay` flag and the `getDisplayWorldHeightAt(coord)` method.
-The flag is used to allow users of the representation to choose between display-scaled
-vs world-scaled sizing.
-
-`getDisplayWorldHeightAt(coord)` will return the world height at a particular point that
-corresponds to the viewport height. In other words, it is the height of the rectangle formed
-by the intersection of the plane (defined by the camera normal and coord) and the view
-frustrum. A typical usage of this method is to compute some fraction of this height and
-set that as the representation scale.
-
-Look at the `SphereHandleRepresentation` as an example for how `getDisplayWorldHeightAt`
+In order to scale a representation such that it retains the same size in
+display space, qt must use the `scaleByDisplay` and `scaleInPixels` flag and
```

Comment:
Is this a qt reference? If this is just a reference, it would help if it is noted as such, or just remove it altogether for clarity.

Reply:
Oh that's a typo. Not sure how "qt" got in there haha.